src/lib.rs

```diff
@@ -660,6 +660,18 @@ pub enum ValidationError {
     MissingModuleForCommand(String, String),
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
+#[serde(rename_all = "kebab-case")]
+pub struct Wapm {
+    pub namespace: String,
+    pub package: Option<String>,
+    pub wasmer_extra_flags: Option<String>,
+    pub abi: Abi,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub fs: Option<HashMap<String, PathBuf>>,
+    pub bindings: Option<Bindings>,
+}
+
```

Comment:
Can you move this into a separate `wapm_toml::rust` module? I'd like to make sure there is a clean boundary between the "main" `wapm.toml` format and the bits that are `cargo wapm`-specific.

Reply:
ok done